acceptable/_service.py

```diff
@@ -110,32 +117,31 @@ def groups(self):
 
     def serialize(self):
         """Serialize into JSONable dict, and associated locations data."""
-        api_metadata = {
-            # $ char makes this come first in sort ordering
-            '$version': self.current_version,
-        }
+        api_metadata = OrderedDict()
+        # $ char makes this come first in sort ordering
+        api_metadata['$version'] = self.current_version
```

Comment:
$ prefix is unnecessary now?

Reply:
Yes, but I still like the indication that it's special